manifests/metallb.yaml

```diff
@@ -129,6 +537,32 @@ rules:
   - podsecuritypolicies
   verbs:
   - use
+- apiGroups:
+  - metallb.io
+  resources:
+  - addresspools
+  verbs:
+  - create
+  - get
+  - list
+  - watch
+- apiGroups:
+  - metallb.io
+  resources:
+  - bfdprofiles
+  verbs:
+  - create
+  - get
+  - list
+  - watch
+- apiGroups:
+  - metallb.io
+  resources:
+  - bgppeers
+  verbs:
+  - get
```

Comment:
can you also provide `create` permission for `bgppeers` ?

Reply:
we would we need them?

Reply:
@fedepaol please ignore this comment, this was a wrong assumption while testing operator patch.